src/modules/QtGraphicalEffects/RectangularGlow.js

```diff
@@ -0,0 +1,92 @@
+QmlWeb.registerQmlType({
+  module: "QtQuick",
+  name: "RectangularGlow",
+  versions: /.*/,
+  baseClass: "Item",
+  properties: {
+    cached: {
+      type: "bool",
+      initialValue: false
+    },
+    color: {
+      type: "color",
+      initialValue: "white"
+    },
+    cornerRadius: {
+      type: "real",
+      initialValue: 0
+    },
+    // This property defines how many pixels outside the item area are reached by the glow.
+    glowRadius: {
+      type: "real",
+      initialValue: 0
+    },
+    spread: {
+      type: "real",
+      initialValue: 0
+    },
+  }
+}, class {
+  constructor(meta) {
+    QmlWeb.callSuper(this, meta);
+
+    const bg = this.impl = document.createElement("div");
+    bg.style.pointerEvents = "none";
+    bg.style.position = "absolute";
+    bg.style.left = bg.style.right = bg.style.top = bg.style.bottom = "0px";
+    bg.style.border = "none";
+    bg.style.backgroundColor = "black";
+    this.dom.appendChild(bg);
+
+    this.colorChanged.connect(this, this.$onColorChanged);
+    this.glowRadiusChanged.connect(this, this.$updateBoxShadow);
+    this.cornerRadiusChanged.connect(this, this.$updateBoxShadow);
+    this.widthChanged.connect(this, this.$updateBoxShadow);
+    this.heightChanged.connect(this, this.$updateBoxShadow);
+    this.spreadChanged.connect(this, this.$onSpreadChanged);
+  }
+  $onColorChanged(newVal) {
+    this.impl.style.backgroundColor = new QmlWeb.QColor(newVal);
```

Comment:
`new QmlWeb.QColor` is not required, `newVal` is already an instance of `QColor` here.